Makefile

```
@@ -683,6 +684,20 @@ test-capd-junit: $(SETUP_ENVTEST) $(GOTESTSUM) ## Run unit and integration tests
 	$(GOTESTSUM) --junitfile $(ARTIFACTS)/junit.infra_docker.xml --raw-command cat $(ARTIFACTS)/junit.infra_docker.stdout
 	exit $$(cat $(ARTIFACTS)/junit.infra_docker.exitcode)
 
+.PHONY: test-runtime-sdk-test-extension
+test-runtime-sdk-test-extension: $(SETUP_ENVTEST) ## Run unit and integration tests for the test extension
```

```suggestion
test-test-extension: $(SETUP_ENVTEST) ## Run unit and integration tests for the test extension
```
WDYT about dropping "runtime-sdk"? Just makes it shorter and in other places we just call it test extension

---

I just hate the `test-test` repetition, but I'm happy to change it if you have a strong preference?

---

Just feels strange to include some prefix just to avoid test-test :D

---

I've updated it now - maybe the better discussion would be to change `test-extension` 😆, but the naming is no blocker for me here.

---

Thank you!